### DISCUSSION TITLE
add option in toml file to show status screen when kodi idle

### DISCUSSION THREAD

**Discussion by nico1080** [mattblovell/kodi_panel#15] [Ideas]
I don't use the touch interupt feature, so in my fork I commented some line to have this behaviour https://github.com/mattblovell/kodi_panel/commit/e2048dcebe81d64198d9a24948dcbe0eab19c42f

Having a "status screen when kodi is idle" setting in the toml file would allow user to have this behaviour

**Comment by mattblovell** [comment]
I agree this should be a straightforward change.

**Comment by mattblovell** [comment]
Should be possible with v1.22
